source/subs_functions/subs_functions.bas

```diff
@@ -1427,6 +1427,55 @@ id.ret = LONGTYPE - ISPOINTER
 id.hr_syntax = "_PRINTMODE[(imageHandle&)]"
 regid
 
+' a740g: Extended Font support
+clearid
+id.n = qb64prefix$ + "UFontHeight"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 1
+id.callname = "func__UFontHeight"
+id.args = 1
+id.arg = MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "[?]"
+id.ret = LONGTYPE - ISPOINTER
+id.hr_syntax = "_UFONTHEIGHT&[(fontHandle&)]"
+regid
+
+clearid
+id.n = qb64prefix$ + "UPrintWidth"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 1
+id.callname = "func__UPrintWidth"
+id.args = 3
+id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "?[,[?][,[?]]]"
+id.ret = LONGTYPE - ISPOINTER
+id.hr_syntax = "_UPRINTWIDTH&(text$[, utfEncoding&][, fontHandle&])"
+regid
+
+clearid
+id.n = qb64prefix$ + "ULineSpacing"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 1
+id.callname = "func__ULineSpacing"
+id.args = 1
+id.arg = MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "[?]"
+id.ret = LONGTYPE - ISPOINTER
+id.hr_syntax = "_ULINESPACING&[(fontHandle&)]"
+regid
+
+regid
+clearid
+id.n = qb64prefix$ + "UPrintString"
+id.Dependency = DEPENDENCY_LOADFONT
+id.subfunc = 2
+id.callname = "sub__UPrintString"
+id.args = 6
+id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "(?,?),?[,[?][,[?][,?]]]"
+id.hr_syntax = "_UPRINTSTRING (x&, y&), text$[, maxWidth&][, utfEncoding&][, fontHandle&]"

```

Comment:
The brackets at the end should reflect the `specialformat`.

Reply:
This one too due to the simplified help texts used by previous authors.